app/controllers/pages_controller.rb

```diff
@@ -10,6 +10,10 @@ def cn2021_cn2022
     @chapters = Chapter.all
   end
 
+  def changes_999l
+    @disable_switch_service_banner = true
```

Comment:
usually we put  disable switches on the before action:
` before_action :disable_switch_service_banner, only: [:changes_999l]`

Reply:
Yeah makes sense!